app/src/org/commcare/android/database/DbUtil.java

```diff
@@ -153,71 +152,6 @@ public void postKey(SQLiteDatabase database) {
        oldDb.close();
    }
    
-   /**
-    * Static experimentation code. Isnt for prod, but left in as an example and location to
-    * test functionality
-    */
-   public static void getGroups() {
```

Comment:
This was unused and undocumented, so it didn't look of use anymore.
If you think it should stick around, please document what it does.